app/src/ui/accessibility/aria-live-container.tsx

```diff
@@ -69,14 +69,32 @@ export class AriaLiveContainer extends Component<
     this.onTrackedInputChanged.cancel()
   }
 
+  /** If the children has a tab focusable element such as a link, then we get
+   * hidden tabs on the screen. For screen readers, we only need the text */
+  private getChildrenAsText(children: React.ReactNode): string {
```

Comment:
This stands out as not very idiomatic to me. Iterating over the (react) children is usually a good way to get into trouble as it's operating over the unprocessed JSX tree and not what would eventually get rendered.

It seems to me that what we're wanting to do here is restrict the AriaLive component to only text (which makes sense to me) then we should make it accept a required `message` prop of type `string` and stop rendering children instead of trying to poke at the child innards.

Reply:
Done. https://github.com/desktop/desktop/pull/16993/commits/d2deac26404c8552ca346c2935f8a10b82bcebbe Tho, this does make the components be a lot more verbose than needed most of the time. :/ Not a huge fan of that, but, I think it does make more sense than iterating over the children (which seems discouraged). 

I thought of doing a hybrid approach that says if a message is provided, use that otherwise use the children so that we only have to provide a message if the children have tab content... but that seems fraught for someone to make the mistake of having an invisible tab which is not obvious to detect. Plus.. just more code maintenance.

Reply:
BTW, I did regression retest along with some known edge case, 
- Autocompletion results
- changed list status
- filter list results
- add repositories errors
- tooltipped content thing
- create repository error
- git warning.